react/features/base/config/functions.web.ts

```diff
@@ -58,7 +58,7 @@ export function getWebHIDFeatureConfig(state: IReduxState): boolean {
 /**
  * Checks if the specified button is enabled.
  *
- * @param {string} buttonName - The name of the button.
+ * @param {string} buttonName - The name of the button
```

Comment:
I think this should be: `@param {string} buttonName - The name of the button. See: {@link interfaceConfig}.` adn the linter should be happy.